pkgs/top-level/static.nix

```diff
@@ -276,4 +280,28 @@ in {
   libev = super.libev.override { static = true; };
 
   libexecinfo = super.libexecinfo.override { enableShared = false; };
+
+  xorg = super.xorg.overrideScope' (xorgself: xorgsuper: {
```

Comment:
I think it would be preferably to introduce some kind of `enableStatic` option in the xorg derivations, if possible, but I guess that is difficult because they are "generated" and then overridden.

Reply:
I don't think we can change the signature of a function while overriding...

In any case I'd rather see something like https://github.com/NixOS/nixpkgs/issues/83664 implemented.

Reply:
It's possible with something like
```
libXxf86dga = lib.makeOverrideable ({enableStatic, ... } @attrs: super.libXxf86dga.overrideAttrs(oldAttrs: {
} // attrs));
```
but yea, not really worth it.